schema/repository.yaml

```diff
@@ -2,24 +2,29 @@ title: Repository
 description: Repository defines a remote git repository.
 type: object
 properties:
-  branch:
-    description: Branch provides repository branch.
-    type: string
   connector:
     description: Connector provides the repository connector.
     type: string
   name:
     description: Name provides the repository name.
     type: string
-  pr:
-    description: PR provides the repository pull request.
+  depth:
```

Comment:
hmm, `depth` really belong under clone since this is a clone setting.  Is there a reason for moving this to `repository`?  maybe we need to revisit the structure before we proceed.